src/components/Dashboard/TodoList.vue

```diff
@@ -30,10 +30,7 @@
           'pay-error': isStatusDraft(item) && isPayError(item)
         }"
       >
-        <v-expansion-panel-header
-          class="no-dropdown-icon pa-0"
-          :class="{'invalid-section': isTypeAlteration(item) && requiresAlteration && !item.goodStanding}"
```

Comment:
This class didn't display correctly so I deleted it. This will be reviewed and perhaps updated in the near future when we talk about BC/CCC/ULC alterations vs alter-to-BEN filings.